### DISCUSSION TITLE
Flows [Script]: is it possible to make an http call?

### DISCUSSION THREAD

**Discussion by dkamali-mc** [directus/directus#15908] [Archive]
using the script function in Flows, I would like to loop over an array of objects and make an outbound HTTP call to a third-party service. 

is that possible? I would appreciate it if someone can share an example

Thanks

**Comment by br41nslug** [comment]
You can pass the array to a "trigger flow" operation which will run the other flow for each item in the array.

**Comment by ponceleon** [comment]
Thanks, this saved my day.

I share a brief example in case someone else finds it useful.

```js
module.exports = async function(data) {

   if( 1 == 2 ) { return "true string or anything" }
   else { throw 'Parameter is false!' }  // intensional throw error
   
}
```
